.github/workflows/gui-pull-request.yml

```diff
@@ -0,0 +1,64 @@
+# This file is not auto-generated. Feel free to edit it.
+
+name: ✨ GUI Pull Request
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # Read-only access to repository contents
+  issues: write # Write access to issues
+  pull-requests: write # Write access to pull requests
+  statuses: write # Write access to commit statuses
+  checks: write
+
+jobs:
+  changed-files:
+    runs-on: ubuntu-latest
+    name: 🔍 Detect changed files in GUI
+    outputs:
+      all_changed_files: ${{ steps.changed-files.outputs.all_changed_files }}
+      any_changed: ${{ steps.changed-files.outputs.any_changed }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v44
+        with:
+          files: |
```

Comment:
what about `Cargo.*`, `pnpm*`, `corepack*`, `package.json` and so on?

Reply:
good catch. not sure about the cargo stuff though, bup updates for lock file/lint configs and like should be listed